### DISCUSSION TITLE
useMount为什么是用的LaunchedEffect而不是SideEffect？

### DISCUSSION THREAD

**Discussion by danatechgithub4** [junerver/ComposeHooks#6] [Q&A] [answered]
RT

useUnmount用的是DisposableEffect

**Comment by junerver** [comment, marked as answer]
> RT
> 
> useUnmount用的是DisposableEffect

因为很常见的一个需求是在组件mount时执行一些挂起函数，例如网络请求，这个钩子被设计成只执行一次，所以不可以使用SideEffect
